lib/functions.php

```diff
@@ -16,6 +16,10 @@ function createModuleRegistry() {
 	}
 	$registry['blueprints']['pages/modules'] = [
 		'title' => 'Modules',
+		'options' => [
```

Comment:
Is it actually necessary to add options to the modules blueprint? I use the custom redirect field so you can't even access this page in the panel.

Reply:
![Screenshot 2020-09-19 at 21 41 55](https://user-images.githubusercontent.com/4954323/93687816-123f6c80-fac1-11ea-896c-f48947ed6e19.png)
Yes because its accesible from pages section. In fact i think all options should disabled not only the ones i mentioned.

Reply:
I don't think the modules container should be accessible tbh. While there's not `hide` option (See https://kirby.nolt.io/23) I normally define the templates for subpages so the modules container isn't listed.